chia/consensus/blockchain.py

```diff
@@ -242,6 +244,8 @@ async def receive_block(
             fork_point_with_peak,
             self.get_block_generator,
         )
+        done = time.time()
+        log.debug(f"validate_block_body took {done-start} seconds")
```

Comment:
You might want to differentiate between the two calls, so we can see the differences.

Reply:
done